> Where is actual error msg?

```
TrinityCore/src/server/game/Scripting/ScriptReloadMgr.cpp: In member function ‘void HotSwapScriptReloadMgr::ProcessLoadScriptModule(const boost::filesystem::path&, bool)’:
TrinityCore/src/server/game/Scripting/ScriptReloadMgr.cpp:875:35: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  875 |                 std::this_thread::sleep_for(std::chrono::seconds(5));
      |                                   ^~~~~~~~~
TrinityCore/src/server/game/Scripting/ScriptReloadMgr.cpp:892:31: error: ‘sleep_for’ is not a member of ‘std::this_thread’
  892 |             std::this_thread::sleep_for(std::chrono::seconds(5));
      |                               ^~~~~~~~~
```

--- a/src/server/game/Scripting/ScriptReloadMgr.cpp
+++ b/src/server/game/Scripting/ScriptReloadMgr.cpp
@@ -56,9 +56,9 @@ ScriptReloadMgr* ScriptReloadMgr::instance()
 #include <algorithm>
 #include <fstream>
 #include <future>
-#include <thread>
 #include <memory>
 #include <sstream>
+#include <thread>
 #include <type_traits>
 #include <unordered_map>
 #include <unordered_set>

